spec/rails_helper.rb

```diff
@@ -0,0 +1,50 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```

Comment:
Let's get rid of all the comments here. They might seem helpful at first but they will become out of date and misleading. A good thing to do is link to the documentation for configuration at the top:

```ruby
# https://relishapp.com/rspec/rspec-core/v/3-1/docs/configuration
```